src/utils/googleMaps/markers/poi/index.ts

```diff
@@ -40,3 +40,15 @@ export const clear = (): void => {
 
   markerRef.current = {};
 };
+
+export const setIcon = (
```

Comment:
Maybe change to `setHightlightIcon`  will be better?
Because this function only use in highlight case, separate from others' situation.

Reply:
I rename it to `toggleHighlightIcon`
Thanks for feedback~